.github/workflows/release.yaml

```diff
@@ -181,8 +181,22 @@ jobs:
         git commit -m "Release $RELEASE_VERSION"
         msg="Release ${{ env.RELEASE_VERSION }}"
         git tag --annotate --message "${msg}" ${{ env.RELEASE_VERSION }}
-        # push both the tag as well as a release branch with that tag.
+        # push the tag.
         git push origin ${{ env.RELEASE_VERSION }}
+    # in case we do not have a release candidate, we should merge the release into the branch
+    # this is so that the tag shows up in the branch and we can use it for the release notes.
```

Comment:
This sound more complicated that is should be. What do we merge in which branch?